ref/netstandard1.3/Microsoft.Build/Microsoft.Build.cs

```diff
@@ -1327,6 +1327,29 @@ public enum TargetResultCode : byte
 }
 namespace Microsoft.Build.Logging
 {
+    public partial class BinaryLogger : Microsoft.Build.Framework.ILogger
```

Comment:
You'll need to build against CoreCLR to regenerate these public APIs.

`cibuild.cmd --build-only --target CoreCLR`